src/frontend/templates/pages/networks.html

```diff
@@ -1,57 +1,61 @@
-  again is there anyone who loves or pursues or desires to obtain pain of itself, because it is pain, but because
-  occasionally circumstances occur in which toil and pain can procure him some great pleasure. To take a trivial
-  example, which of us ever undertakes laborious physical exercise, except to obtain some advantage from it? But who has
-  any right to find fault with a man who chooses to enjoy a pleasure that has no annoying consequences, or one who
-  avoids a pain that produces no resultant pleasure?"
-  <h1 class="title">Section 1.10.33 of "de Finibus Bonorum et Malorum", written by Cicero in 45 BC</h1>
-
-  "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque
-  corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui
-  officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita
-  distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime
-  placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut
-  officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae.
-  Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut
-  perferendis doloribus asperiores repellat."
-  <h1 class="title">1914 translation by H. Rackham</h1>
-
-  "On the other hand, we denounce with righteous indignation and dislike men who are so beguiled and demoralized by the
-  charms of pleasure of the moment, so blinded by desire, that they cannot foresee the pain and trouble that are bound
-  to ensue; and equal blame belongs to those who fail in their duty through weakness of will, which is the same as
-  saying through shrinking from toil and pain. These cases are perfectly simple and easy to distinguish. In a free hour,
-  when our power of choice is untrammelled and when nothing prevents our being able to do what we like best, every
-  pleasure is to be welcomed and every pain avoided. But in certain circumstances and owing to the claims of duty or the
-  obligations of business it will frequently occur that pleasures have to be repudiated and annoyances accepted. The
-  wise man therefore always holds in these matters to this principle of selection: he rejects pleasures to secure other
-  greater pleasures, or else he endures pains to avoid worse pains."
+{% if not run %}
+No currently active run to view networks for.
+{% elif viewing_current_run %}
+<h1 class="title">Networks for Current Run ({{run.name}}) </h1>
+{% else %}
+<h1 class="title">Networks for {{run.name}}</h1>
+{% endif %}
+
+{% if run and object_list|length <= 0 %}
+No networks found for this run!
```

Comment:
`I think it's cleaner to have fewer variables and less action-at-a-distance in the code. 
The `object_list` (now renamed to `networks_list`) networks list is exactly a variable that is looped over just below in the same template. So it makes sense to have the check on that variable here, instead of having to know that `run_has_no_network` is equivalent to `networks_list` being length 0.